package.json

```diff
@@ -33,6 +33,7 @@
     "@probe.gl/bench": "^3.0.2",
     "@probe.gl/test-utils": "^3.0.2",
     "babel-plugin-transform-builtin-extend": "^1.1.2",
+    "coveralls": "^3.0.0",
```

Comment:
Assume `coveralls` cannot be installed by a dependency?

Reply:
Hmm we can, though a repo might want to choose a different coverage tracking service?